src/ui/utils/key-shortcuts.ts

```diff
@@ -73,13 +73,17 @@ export function cancelBubbling(listener: KeyboardEventListener): KeyboardEventLi
   };
 }
 
+function isEditableTag(target: HTMLElement) {
+  return (
+    ["INPUT", "TEXTAREA"].includes(target.tagName) && !(target.getAttribute("readonly") !== null)
+  );
```

Comment:
wow - this is super smart. I wouldn't have thought of going into `key-shortcuts`, but it's obvious in hindsight

Reply:
Also I'm just realizing this double negation is completely unreadable, what was I thinking? 😆